cluster/images/stack-aws/Dockerfile

```diff
@@ -4,9 +4,9 @@ RUN apk --no-cache add ca-certificates bash
 ARG ARCH
 ARG TINI_VERSION
 
-ADD crossplane /usr/local/bin/
+ADD stack /usr/local/bin/
```

Comment:
https://github.com/crossplaneio/stack-azure/pull/19#issuecomment-533741656

There's been some discussion about what Stack binaries should be named in the above thread, but I don't think we've reached a great direction.

Reply:
I'm really reluctant to bike shed the name of these binaries, but @suskin's point about not using the packaging format (stacks) in the binary name resonated with me a little. Happy to merge this as is if you feel strongly that `stack` is the best name here.

Reply:
@negz I talked to @suskin and made a comment in the above link.